zmon_worker_monitor/zmon_worker/notifications/hipchat.py

```diff
@@ -19,8 +21,15 @@ def notify(cls, alert, *args, **kwargs):
 
         color = 'green' if alert and not alert.get('is_alert') else kwargs.get('color', 'red')
 
+        zmon_host = kwargs.get('zmon_host', cls._config.get('zmon.host'))
+        alert_id = alert['alert_def']['id']
+        alert_url = urlparse.urljoin(zmon_host, '/#/alert-details/{}'.format(alert_id)) if zmon_host else ''
+        message_text = cls._get_subject(alert, custom_message=kwargs.get('message'))
+        if kwargs.get('link', False):
+            message_text += ' -- <a href="{}">GO TO ALERT</a>'.format(alert_url)
```

Comment:
probably lower case, this is "too loud" IMO

Reply:
and add a `link_text` in kwargs which defaults to "go to alert"

Reply:
@elauria should the `<a href=` also have a `target="_blank"` (or what ever is needed to open in a new tab)?